src/main/resources/com/ibm/devops/connect/DevOpsGlobalConfiguration/config.jelly

```diff
@@ -41,7 +41,7 @@
             <c:select/>
         </f:entry>
         <f:advanced>
-            <f:entry title="${%Api Token}" field="apiToken">
+            <f:entry title="${%User Access Key}" field="apiToken">
```

Comment:
I think we want to remove this from the `advanced` section and put it in the main section, since this now seems required for most calls